<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewSettingsWidget</class>
 <widget class="QWidget" name="ViewSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="config_tab">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <widget class="QCheckBox" name="showTagHistory">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>109</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Tag History</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="tagHistoryN">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="LogInterval">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Log interval</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="LogIntervalSB">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="SendataIntervalSB">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="SendData">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Send Data</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>210</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Minuten</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>x Geschiedenis</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Object om te zoeken:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>230</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Geen log, Geen Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alleen Log</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alleen Debug (QTSource)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Log,Debug (QTSource)</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>M , tollerantie</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Log Modus:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbAlarm">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Bril alarm</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="floorplan_tab">
      <attribute name="title">
       <string>Floor Plan</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>pixels</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="floorplanXScale_sb">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>pixesl/m</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X Offset</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="floorplanXOff_sb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>-65536</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="floorplanPath_lb">
         <property name="text">
          <string>No Floor Plan</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="floorplanFlipY_cb">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>pixels</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="floorplanYScale_sb">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>pixels/m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="floorplanYOff_sb">
         <property name="minimum">
          <number>-65536</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>X Scale</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Flip X</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Y Offset</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="floorplanFlipX_cb">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Y Scale</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="floorplanOpen_pb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Flip Y</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="showOrigin">
           <property name="text">
            <string>Show Origin</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="scaleY_pb">
           <property name="text">
            <string>Y Scale</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="scaleX_pb">
           <property name="text">
            <string>X Scale</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="origin_pb">
           <property name="text">
            <string>Set Origin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="grid_tab">
      <attribute name="title">
       <string>Grid</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>21</y>
         <width>132</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Width (m)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="gridWidth_sb">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Height (m)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="gridHeight_sb">
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="gridShow">
          <property name="text">
           <string>Show Grid</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>gridWidth_sb</tabstop>
  <tabstop>gridHeight_sb</tabstop>
  <tabstop>floorplanOpen_pb</tabstop>
  <tabstop>floorplanXOff_sb</tabstop>
  <tabstop>floorplanYOff_sb</tabstop>
  <tabstop>floorplanXScale_sb</tabstop>
  <tabstop>floorplanYScale_sb</tabstop>
  <tabstop>floorplanFlipX_cb</tabstop>
  <tabstop>floorplanFlipY_cb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
